lti_consumer/plugin/views.py

```diff
@@ -101,6 +105,35 @@ def has_block_access(user, block, course_key):
     return course_access and block_access
 
 
+def parse_custom_parameters(block):
```

Comment:
```suggestion
def custom_parameters_parser(block):
```

What do you think about changing the name?